src/city-council-district/city-council-district.repository.ts

```diff
@@ -0,0 +1,24 @@
+import { Inject } from "@nestjs/common";
+import { DB, DbType } from "src/global/providers/db.provider";
+import { DataRetrievalException } from "src/exception";
+import { FindManyRepo } from "./city-council-district.repository.schema";
+
+export class CityCouncilDistrictRepository {
+  constructor(
+    @Inject(DB)
+    private readonly db: DbType,
+  ) {}
+
+  async findMany(): Promise<FindManyRepo> {
+    try {
+      return await this.db.query.cityCouncilDistrict.findMany({
+        columns: {
+          id: true,
+          liFt: false,
```

Comment:
Outta curiosity, why is it that we need to specify false for `liFt` but not for `mercatorFill` or `mercatorLabel` just to get the `id`?

Reply:
It seems that without the `liFt` column, neither the `mercatorFill` or `mercatorLabel` are returned in the response. I am not sure why TBH. We can ask @TangoYankee for an explanation next week.

Reply:
The 'lift: false' is superfluous. specifying only 'id: true' will achieve the same result

Reply:
oh, i'll get rid of it (`liFt`)

Reply:
@pratishta **TIL**: it seems that when you specify what columns you want, the drizzle [docs](https://orm.drizzle.team/docs/rqb#partial-fields-select) state that ONLY those columns are returned. Not only that but when both true and false select options are present, all false options are ignored. So the `liFt: false` is bering ignored.